concepts/observability-strategy/strategy.md

```diff
@@ -1,54 +1,60 @@
 
-## Goals
+![Current observability stack](./assets/strategy-current.drawio.svg "Current observability stack")
 
-### Stages of observability
+- Application Logs
+  - Instrumentation is done by printing logs to stdout/stderr as recommended by Kubernetes best practices, system components are following that guide and logs are available already; no other way of getting logs collected
+  - FluentBit as log collector defines a hardcoded pipeline for collecting, enriching and pushing the logs to a backend; no further configuration of pipelines at runtime, especially additional outputs for external systems cannot be configured
+  - Storage is realized by a lightweight (non-scalable) Loki installation, no configuration at runtime
+  - Reporting is implemented by the log explorer of Grafana
+- Metrics
+  - For instrumentation workload needs to expose metrics in the prometheus-compatible format, system components are already doing that; no other ways of exporting metrics is supported like using the OTLP push based protocol
+  - Collection of metrics from system and custom workloads is done by a lightweight prometheus installation; configuration of collection can be defined at runtime but the storage might not scale in accordingly requiring adjustments at deploy time; configuration of outputs is not possible at runtime and only prometheus-specific protocols are supported (forward or federation)
+  - Prometheus is collecting and storing the metrics; setup is non-scalable and resource settings cannot be configured at runtime
+  - Reporting is done via a Grafana installation which loads pre-bundled dashboards; dashboards can be added at runtime
+- Traces
+  - 
 
-Observability can be split up into:
-1. Instrumentation of the users application
-2. The collection and preprocessing of the signals from the user application and the surrounding infrastructure, including metadata enrichment
-3. The delivery of the signal to an backend for storage
-4. An optional aggregation of the signals
-5. Storage of the signals
-6. The analysis, querying, and dashboarding of the signals
+The current solution provides a feature-rich end-to-end setup at a first glance. However, at a second look users realize soon the major drawbacks and usually will require additional stacks deployed besides:
+- Very limited integration possibilities into external systems; that is usually required for different reasons (cross-cluster correlation, forensic analysis, long time storage,..); integration points are not vendor-neutral
+- Very limited configuration options for data enrichment/filtering; users are interested in specific aspects for specific scenarios
+- Storage backends are non-scalable and with can be used only in limited scenarios. There is no option to upgrade the backend into a scalable setup but also no way to integrate with other solutions
 
-![a](./assets/stages.drawio.svg)
+### Requirements
```

Comment:
I think this should be h2 not h3, otherwise it's part of the "Current situation" section.

```suggestion
## Requirements
```